src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -96,14 +95,15 @@ JsonObject ColorScheme::ToJson() const
     jsonObject.Insert(FOREGROUND_KEY, fg);
     jsonObject.Insert(BACKGROUND_KEY, bg);
  
-	for (int i = 0; i < TABLE_SIZE; i++)
-	{
-		auto current = TABLE_COLORS[i];
-		auto& color = _table[i];
-		auto s = JsonValue::CreateStringValue(Utils::ColorToHexString(color));
-				
-		jsonObject.Insert(current, s);
-	}
+    int i = 0;
+    for (const auto& current : TABLE_COLORS)
+    {
+        auto& color = _table[i];

```

Comment:
Is `_table` an STL collection here? We should probably use `_table.at(i)` instead so it will throw if you are out of bounds then.

Reply:
And below probably.

Reply:
🐰 

I was copying the legacy code which was using `_table[i]`, so I've updated that too for consistency

Reply:
Good call on copying the pattern. This is just one of those things that we've been on the lookout for.